examples/catalog-module/index.js

```diff
@@ -11,7 +11,7 @@ ReactDOM.render(
         path: '/',
         title: 'Foo',
         imports: {Foo: require('./components/Foo/Foo')},
-        component: require('./components/Foo/Foo.docs.md')
+        component: ContentLoader('./components/Foo/Foo.docs.md')
```

Comment:
I don't think that'll work (ContentLoader tries to fetch the URL, `require` imports it as a module)?